roles/manage-gcp-infra/tasks/create-infrastructure.yaml

```diff
@@ -0,0 +1,45 @@
+---
+- name: setup gcloud configuration
+  shell: gcloud config configurations create openshift-{{ gcloud_env }}
+  register: result
+  failed_when: not ((result.rc == 0) or ("it already exists" in result.stderr))
+  changed_when: not ("it already exists" in result.stderr)    
+  
+- name: setup gcloud authenticationn
+  shell: gcloud auth activate-service-account --key-file {{ credentials_file }} --project {{ gcloud_project_name }}
+
+- name: create a temp directory fror the google templates
+  file:
```

Comment:
Same as above, please use the `tempfile` module

Reply:
Also make sure to register a handler to remove it when done

Reply:
done using the tempfile module.
can you show me how to use a handler?

Reply:
Here's one example:
https://github.com/redhat-cop/infra-ansible/blob/master/roles/haproxy-config/tasks/prereq.yml#L10

... with the corresponding handler:

https://github.com/redhat-cop/infra-ansible/blob/master/roles/haproxy-config/handlers/main.yml#L3-L6

Reply:
done